src/Microsoft.AspNet.OData.Shared/IEdmChangedObjectOfT.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.OData
+{
+    /// <summary>
+    /// Base interface to be implemented by any Delta object required to be part of the DeltaFeed Payload.
+    /// </summary>
+    /// <typeparam name="TStructuralType"></typeparam>
+    public interface IEdmChangedObject<TStructuralType> : IEdmChangedObject
```

Comment:
>TStructuralType [](start = 39, length = 15)

It seems we need use the generic type in this class? So, what's the design pattern?